sw/airborne/modules/optical_flow/px4flow.c

```diff
@@ -40,11 +40,18 @@ bool_t optical_flow_available;
 
 // request struct for mavlink decoder
 struct mavlink_msg_req req;
+#define PIX4FLOWSENDER_ID 1333
```

Comment:
ABI sender id should be in subsystems/abi_sender_ids.h file and should be between 0 and 255